bevy_rapier3d/Cargo.toml

```diff
@@ -43,7 +43,7 @@ log = "0.4"
 serde = { version = "1", features = [ "derive" ], optional = true}
 
 [dev-dependencies]
-bevy = { version = "0.11", default-features = false, features = ["x11"]}
+bevy = { version = "0.12.0", default-features = false, features = ["x11", "tonemapping_luts"]}
```

Comment:
Same here

Reply:
And `0.12.0` -> `0.12`

Reply:
"tonemapping_luts" prevents a warning/error from being thrown. (See https://github.com/bevyengine/bevy/pull/10253)

> And `0.12.0` -> `0.12`

I'll clean that up :)